stripe/src/main/java/com/stripe/android/model/SourceParams.java

```diff
@@ -49,6 +50,7 @@
     @IntRange(from = 0) private Long mAmount;
     private Map<String, Object> mApiParameterMap;
     private String mCurrency;
+    @Nullable private String mCustomType;
```

Comment:
Should we name this (and the setter/getter) to `TypeRaw`, to match the Source model?